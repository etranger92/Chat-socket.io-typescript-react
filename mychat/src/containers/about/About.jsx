import React from 'react';
import { Component } from 'react';
import './About.css';

class About extends Component {
  constructor(props) {
    super(props);
    this.state = {};
  }
  render() {
    return (
      <React.Fragment>
        <h1> Hello About</h1>
      </React.Fragment>
    );
  }
}

export default About;
