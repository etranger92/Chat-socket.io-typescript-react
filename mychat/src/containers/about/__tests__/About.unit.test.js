import React from 'react';
import { shallow } from 'enzyme';
import About from '../About';

test('renders HeaderNav', () => {
  const wrapper = shallow(<About />);
  expect(wrapper).toMatchSnapshot();
});
