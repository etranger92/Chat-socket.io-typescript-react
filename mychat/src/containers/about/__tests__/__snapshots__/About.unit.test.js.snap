// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders HeaderNav 1`] = `
<Fragment>
  <h1>
     Hello About
  </h1>
</Fragment>
`;
